projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-product-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,81 @@
+import * as configuration from '../../helpers/product-configuration';
+import * as configurationOverview from '../../helpers/product-configuration-overview';
+import { tabbingOrderConfig as tabConfig } from '../../helpers/accessibility/tabbing-order.config';
+import { verifyTabbingOrder } from '../../helpers/accessibility/tabbing-order';
+
+const testProduct = 'CONF_CAMERA_SL';
+
+const configurator = 'CPQCONFIGURATOR';
+
+const containerSelectorConfigForm = '.VariantConfigurationTemplate';
+const containerSelectorOverviewForm = '[class*=VariantConfig]';
+
+function goToConfigPage(configuratorType, product) {
```

Comment:
To avoid code duplication move methods and constants to the **product-config.ts**

Reply:
Method moved, constants are specific for the test